packages/adminrouter/extra/src/test-harness/modules/generic_test_code/common.py

```diff
@@ -142,6 +150,23 @@ def generic_correct_upstream_request_test(
     assert req_data['request_version'] == http_ver
 
 
+def header_is_absent(headers, header_name):
```

Comment:
❓ Either change this to `assert_ header_is_absent` or make it return a boolean and have the caller `assert header_is_absent...`